scte35/state_test.go

```diff
@@ -253,48 +234,50 @@ func TestDuplicateOut(t *testing.T) {
 func TestOutOutIn(t *testing.T) {
 	st := NewState()
 
-	// 0x34 - 1 - (1/2)
-	open12, e := NewSCTE35(poOpen12)
-	if e != nil {
-		t.Error("NewSCTE35(poOpen12) returned err:", e)
-		t.FailNow()
+	// 0x30
+	padOpenSignalBytes, _ := base64.StdEncoding.DecodeString("/DBsAAH/7m1eAAKQBQb+sX6o+wBWAlRDVUVJAAAAJ3//AAApMuANQAwOQU1DTiBMMDAxMjM0NTYJCFBPOjEyMzQ1DiRiY2IxZGQ4ZS1kMzIzLTQ1ODktOWQ3OC1hM2QxMzYyYWJiYjYwAQGtc8xr")
+	padOpenSignal, err := NewSCTE35(append([]byte{0x0}, padOpenSignalBytes...))
+	if err != nil {
+		t.Errorf("Error creating SCTE-35 signal: %s", err.Error())
 	}
 
-	_, e = st.ProcessDescriptor(open12.Descriptors()[0])
-	if e != nil {
-		t.Error("ProcessDescriptor of out returned unexpected err:", e)
+	_, err = st.ProcessDescriptor(padOpenSignal.Descriptors()[0])
+	if err != nil {
+		t.Error("ProcessDescriptor of out returned unexpected err:", err)
 	}
 	if len(st.Open()) != 1 {
-		t.Error("Unexpected number of open signals after open12 processed.")
+		t.Error("Unexpected number of open signals after padOpenSignal processed.")
 	}
 
-	// 0x34 - 2 - (2/2)
-	open22, e := NewSCTE35(poOpen22)
-	if e != nil {
-		t.Error("NewSCTE35(poOpen22) return err:", e)
+	// 0x34
+	ppoStartSignalBytes, _ := base64.StdEncoding.DecodeString("/DA0AAG3wyKO///wBQb/yRLo+gAeAhxDVUVJQAAAZH/PAAEOFNwICAAAAAArJkYdNAAAMcFkEw==")
+	ppoStartSignal, err := NewSCTE35(append([]byte{0x0}, ppoStartSignalBytes...))
+	if err != nil {
+		t.Error("NewSCTE35(ppoStartSignal) return err:", err)
 		t.FailNow()
```

Comment:
Thoughts on replacing these two lines with `Fatal`?
`t.Fatal("NewSCTE35(ppoStartSignal) return err:", err)`

Reply:
good point. I was doing copy pasta of old tests to write this new one. I like t.Fatal as well.